src/test/java/io/asyncer/r2dbc/mysql/IntegrationTestSupport.java

```diff
@@ -38,10 +39,33 @@ abstract class IntegrationTestSupport {
 
     private final MySqlConnectionFactory connectionFactory;
 
+    private static final MySQLContainer<?> container;
+
     IntegrationTestSupport(MySqlConnectionConfiguration configuration) {
         this.connectionFactory = MySqlConnectionFactory.from(configuration);
     }
 
+    static {
+        String password = System.getProperty("test.mysql.password");
```

Comment:
I believe it's no longer necessary to retrieve the password from system properties. Could we consider removing this line and using a hardcoded password instead?